packages/terra-demographics-banner/lib/DemographicsBanner.scss

```diff
@@ -0,0 +1,89 @@
+@import '~terra-mixins';
+@import './variables';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+
+  @media screen and (max-width: 767px) {
```

Comment:
Let's use a mobile first approach and apply narrow screen styles by default, and then add min-width breakpoint for the `display: flex;` style.

```css
.terra-DemographicsBanner {
  background: $demographics-background;
  color: $demographics-foreground;
  display: block;
  flex-direction: row;
  margin-bottom: 0.714em;
  padding: 0.714em;

  @media screen and (min-width: $terra-small-breakpoint) {
    display: flex;
  }
}
```